docs/glossary.md

```diff
@@ -6,58 +6,60 @@ title: Glossary
 
-**[Bonding](https://blog.nymtech.net/bonding-vs-delegated-staking-in-nym-f11d7adceec3)** - A action for node operators to bond thier mix node or gateway to their Nym Wallet.
+**[Coconut](https://nymtech.net/docs/stable/coconut/)** - A distributed cryptographic signing scheme providing a high degree of privacy for its users.
 
-**Coconut** - A distributed cryptographic signing scheme providing a high degree of privacy for its users.
-
-**CLI** - A command-line interface (CLI) is a text-based user interface (UI) used to run programs, manage computer files and interact with the computer.
+**[CLI](https://nymtech.net/docs/stable/nym-cli/)** - A command-line interface (CLI) is a text-based user interface (UI) used to run programs, manage computer files and interact with the computer.
 
 **Credentials** - Unique values that prove the legitimacy of a user or individual.
 
-**Centralisation** - Refers to the The concentration of control of an activity or organisation under a single authority
+**Centralisation** - Refers to the concentration of control of an activity or organisation under a single authority.
 
-**Delegating NYM** - Similiar to bonding but by non operators and as a proof of their trust on a node operator providing a good quality of service 
+**Delegating NYM** - Similar to bonding but by non-operators and as proof of their trust in a node operator providing a good quality of service. 
 
 **Decentralisation** - Refers to the transfer of control and decision-making from a centralised entity (individual, organization, or group thereof) to a distributed network.
 
 **Encryption** - The process of encoding information.
 
-**[Gateways](https://blog.nymtech.net/gateways-to-privacy-51196005bf5)** - The entrance to the mixnet that encrypted packets must pass through before being forwarded to mixnodes. They also act as a sort of mailbox for messages.
+**[Gateways](https://blog.nymtech.net/gateways-to-privacy-51196005bf5)** - The entrance to the mixnet that encrypted packets must pass through before being forwarded to mix nodes. They also act as a sort of mailbox for messages.
+
+**[Gateway Bonding](https://nymtech.net/docs/stable/run-nym-nodes/nodes/gateways/#bonding-your-gateway)** - A action for node operators to bond their gateway to their Nym Wallet.
+
+**Metadata** - A set of data that describes and gives information about other data such as IP addresses, device types, geolocations etc.
 
-**Metadata** - A set of data that describes and gives information about other data such IP addresses, device types, geolocations etc.
+**[Mixnet](https://nymtech.net/docs/stable/architecture/traffic-flow/)** - Routing protocols that create hard-to-trace communications by using a chain of proxy servers known as mixes which take in messages from multiple senders, shuffle them, and send them back out in random order to the next destination.
```

Comment:
Where is this from? Just a protocol, not protocols. Grab the definition from the network overview page and use that i think